shared/js/activeRoomStore.js

```diff
@@ -857,6 +859,21 @@ loop.store.ActiveRoomStore = (function() {
         actionData.state === SCREEN_SHARE_STATES.ACTIVE);
     },
 
+    receivedCursorPosition: function(actionData) {
```

Comment:
We might want to move the new functionality in this file into a new store, similar to how the `TextChatStore` works.

One advantage of doing this, is that the `RemoteCursorView` could subscribe to this store for the remote cursor position location. Then as I said in my comment for that view, you could just pass in the video size via the props, and have the view calculate the actual position to display the cursor.